src/components/Chips/Chips.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export function Chips({ categoryName }) {
+  return <div class='chip'>{categoryName}</div>;
```

Comment:
here it should be className

Reply:
Thanks for pointing that out. Will correct it.